components/intro-template.tsx

```diff
@@ -18,16 +18,23 @@ export default function IntroTemplate() {
     ? `https://${process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER}.com/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER}/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG}/blob/main/README.md#how-can-i-remove-the-next-steps-block-from-my-blog`
     : `https://github.com/sanity-io/nextjs-blog-cms-sanity-v3#how-can-i-remove-the-next-steps-block-from-my-blog`
 
+  const [hasUTMtags, setHasUTMtags] = useState(false)
+
   useEffect(() => {
     if (typeof window !== 'undefined') {
       setStudioURL(`${window.location.href}studio`)
       setCreatePostURL(
         `${window.location.href}/studio/intent/create/template=post;type=post/`
       )
       setIsLocalhost(window.location.hostname === 'localhost')
+      setHasUTMtags(window.location.search.includes('utm'))
     }
   }, [])
 
+  if (hasUTMtags) {
```

Comment:
I considered adding this in the `index.js` file to make sure this component wasn't being rendered when it isn't needed, but to make it easier for anyone wanting to remove the `intro-template` component from the blog altogether I think it's best to keep the logic all in one place. Thoughts?

Reply:
Agreed, easy to remove is more important 👍